controller/patterns/theater_chase_pattern.py

```diff
@@ -12,12 +12,16 @@ def __init__(self):
 
     def initialize(self):
         self.pattern_segments = []
+        if hasattr(self.params, 'color'):
```

Comment:
Oh, this is totally a bug. We should just rename foreground_color to color in this file

Reply:
Sounds good. Should we standardize whether patterns pull from a `color` rather than a `palette`? Do you think it's easiest to have patterns take a primary color and derive secondary colors from that one?

Reply:
Yeah, would be nice to have one consistent standard so we could use launchpad and DMX to modify the colors of all patterns. Using color would be easiest but what do we do with patterns that using a pallete, eg. fire?

Reply:
I *think* we can use HSV to create color offsets that will replicate the same effect using an arbitrary base color. Not sure, can I give that a shot and you take a look?

Reply:
Sounds great, thank you!